### PR TITLE
HMA-9137 bootstrap upgrade

### PR DESCRIPTION
**Pull request by pdhiman3006**
bootstrap upgrade
- changes dne for HttpCLinetV2 and uuit  test cases and It test cases update

### CODE REVIEW

File: app/uk/gov/hmrc/customerprofile/config/microserviceWiring.scala [hunk 1]
```diff
@@ -1,50 +0,0 @@
...
-import play.api.libs.ws.WSClient
-import uk.gov.hmrc.http.HttpClient
-import uk.gov.hmrc.http.hooks.{HttpHook, HttpHooks}
-import uk.gov.hmrc.play.audit.http.HttpAuditing
-import uk.gov.hmrc.play.audit.http.connector.AuditConnector
-import uk.gov.hmrc.play.http.ws._
-
-import javax.inject.Named
-
-trait Hooks extends HttpHooks with HttpAuditing {
```
Comment (william-hatzar) [inline app/uk/gov/hmrc/customerprofile/config/microserviceWiring.scala:32]: Why was this file removed?

File: app/uk/gov/hmrc/customerprofile/connector/ApplePassConnector.scala [hunk 2]
```diff
@@ -21,15 +21,16 @@ import com.google.inject.name.Named
...
-import uk.gov.hmrc.http.{HeaderCarrier, HttpClient, HttpException, HttpResponse}
+import uk.gov.hmrc.http.{HeaderCarrier, HttpException, HttpResponse, StringContextOps}
 import uk.gov.hmrc.http.HttpReads.Implicits._
+import uk.gov.hmrc.http.client.HttpClientV2
 
 import scala.concurrent.{ExecutionContext, Future}
 
 class ApplePassConnector @Inject() (
-  http:                                                          HttpClient,
+                                     http: HttpClientV2,
```
Comment (william-hatzar) [inline app/uk/gov/hmrc/customerprofile/connector/ApplePassConnector.scala:33]: The formatting here is off, and appears to be for the majority of the files that have been edited. If you need any help getting the formatter configured on your machine give me a shout

File: app/uk/gov/hmrc/customerprofile/config/microserviceWiring.scala (hunk 1 above)
Comment (pdhiman3006) [reply app/uk/gov/hmrc/customerprofile/config/microserviceWiring.scala:32]: Not Needed anymore after HttpClientV2 injection

**Review by william-hatzar — APPROVED** [review]
